sky/clouds/aws.py

```diff
@@ -110,6 +110,23 @@ def get_default_ami(cls, region_name: str, instance_type: str) -> str:
                 }
                 assert region_name in amis, region_name
                 return amis[region_name]
+            if acc_name == 'Inferentia':
+                # Deep Learning AMI Fetched with:
+                # aws ec2 describe-images --region {region} --owners amazon \
+                # --filters 'Name=name,Values=Deep Learning AMI (Ubuntu 18.04) \
+                # Version ??.?' 'Name=state,Values=available' \
+                # --query 'reverse(sort_by(Images, &CreationDate))[:1].ImageId'\
+                # --output text
+                # Inferentia accelerator requires original deep learning AMI
+                amis = {
+                    'us-east-1': 'ami-008a77bad7b109354',
+                    'us-east-2': 'ami-0ae37980d60476156',
+                    # This AMI is 20210623 as aws does not provide a newer one.
+                    'us-west-1': 'ami-0b3e9bf6217f84af6',
+                    'us-west-2': 'ami-08c47b9efd307c86a',
```

Comment:
Is there AMI with Ubuntu 20.04 for inferentia? or 18.04 is the latest?

Reply:
I just updated it to Ubuntu 20.04, but it only has base version without tensorflow or pytorch installed (the multi-platform DLAMI only supports 18.04). Will add some extra setups after adding the examples in this PR. : )